src/test/java/net/masterthought/cucumber/generators/AbstractPageTest.java

```diff
@@ -64,7 +64,7 @@ public void generateReport_DisplaysContentAsEscapedText() {
         assertThat(document.getFeature().getDescription())
                 .isEqualTo("As an Account Holder I want to withdraw cash from an ATM,<br>so that I can get money when the bank is closed");
         assertThat(document.getFeature().getElements()[0].getStepsSection().getSteps()[5].getEmbedding()[3].text())
-                .isEqualTo("Attachment 4 (HTML) <i>Hello</i> <b>World!</b>");
+                .isEqualTo("Attachment 4 (HTML)");
```

Comment:
The element no longer contains any text. It is all contained in the srcDoc attribute of the iframe which we test in mbeddingAssertion.java